dist/_patterns/01-components/01-page-elements/hero.twig

```diff
@@ -0,0 +1,10 @@
+<div class="hero">
+  <div class="hero__content" role=”banner”>
+    <h1 class="hero__header">{{ pageTitle }}</h1>
+    <h2 class="hero__subheader">{{ pageSubTitle }}</h2>
+    <p class="hero__text">{{ summary }}</p>
+    <div class="hero__cta">
+      <button class="button">Hero Button</button>
```

Comment:
The hero button text should be a variable that we can control when including this pattern.

Also, it’d be nice if we could optionally add classes to the button component here when including this pattern. That might be too much complexity though…